src/status_im2/subs/chat/chats.cljs

```diff
@@ -20,6 +20,14 @@
  (fn [chats [_ chat-id]]
    (get chats chat-id)))
 
+(re-frame/reg-sub
+ :community-id-by-chat-id
+ :<- [:chats/chats]
```

Comment:
You can also re-use `chats/chat` and extract the `community-id` in this sub, something like this should work I believe:

```
(re-frame/reg-sub
  :community-id-by-chat-id
  (fn [[_ chat-id]]
    [(re-frame/subscribe [:chats/chat chat-id])])
  (fn [[chat]]
    (:community-id chat)))
```